### ISSUE TITLE
TS -> JS?

### ISSUE THREAD

**Issue by northeastprince** [hotwired/stimulus#733]
Are there plans to drop TypeScript from Stimulus since Turbo did so?

**Comment by dhh** [comment]
Yes, Stimulus will eventually get off TypeScript as well. But we are working on getting Turbo 8 out the door first.
